WordPress/gutenberg
# Update spacing sizes control metrics and icons

## What?

- Improve the icons so they are more visible (higher contrast and not blurry on high-resolution screens) and a bit smaller, to fit better in the Inspector. The color is technically darker than the track value, but optimally it does not look so (as it's smaller visually). 
- Condenses a bit as well, so the controls take up less space — but are still approachable. 
- Adjusts the icons iso they are in line with the label. 
- Small refinement to the settings icon. 

## Testing Instructions

1. Open a post or page.
2. Insert a group block. 
3. Open the block inspector.
4. See Dimensions panel. 
5. Add various padding and margin tools. 

## Screenshots or screencast 

| Before  | After |
| ------------- | ------------- |
|<img width="280" alt="CleanShot 2023-09-14 at 13 18 48" src="https://github.com/WordPress/gutenberg/assets/1813435/5d3b56db-2161-4bfc-bd17-b257c8dd9fd7"><img width="280" alt="CleanShot 2023-09-14 at 13 18 34" src="https://github.com/WordPress/gutenberg/assets/1813435/a74d7b3a-de8c-4c94-ab40-af7663c73e93"><img width="279" alt="CleanShot 2023-09-14 at 13 19 35" src="https://github.com/WordPress/gutenberg/assets/1813435/cd212999-1d2b-4ac8-8174-c156c2fec477"><img width="280" alt="CleanShot 2023-09-14 at 13 29 03" src="https://github.com/WordPress/gutenberg/assets/1813435/9fc9f71a-4953-4b26-8959-4aec974c9db6">|<img width="280" alt="CleanShot 2023-09-14 at 13 20 14" src="https://github.com/WordPress/gutenberg/assets/1813435/6f0eb7e9-3aa3-45b4-8c6d-8466c1a8b669"><img width="280" alt="CleanShot 2023-09-14 at 13 20 59" src="https://github.com/WordPress/gutenberg/assets/1813435/f2de1b9b-29af-4966-b871-b830e9c3ed69"><img width="280" alt="CleanShot 2023-09-14 at 13 22 25" src="https://github.com/WordPress/gutenberg/assets/1813435/5600a4aa-8a97-46c2-b69e-19befdebf871"><img width="279" alt="CleanShot 2023-09-14 at 13 28 57" src="https://github.com/WordPress/gutenberg/assets/1813435/f0cdb60b-f11e-421e-9239-7b8b117d5280">|
